### PR TITLE
Firebase code refactor

### PR DESCRIPTION
**Pull request by gtqifan**
Key Update:
- Firebase code refactor

### CODE REVIEW

**Comment by gtqifan** [comment]
@Jennywh please review this one

File: src/screens/LoginScreen.js [hunk 1]
```diff
@@ -25,21 +26,29 @@ const LoginScreen = () => {
...
+          // Navigate to the main app screen after successful login
+          navigation.navigate('MainApp');
+        } else {
+          // Navigate to the profile creation page if no profile exists
+          navigation.navigate('ProfileCreation');
+        }
+
       } else {
-        setError('Please verify your email before logging in. Remember to check your inbox and junk folder.');
+        setError('Please verify your email before logging in. Check your inbox and junk folder.');
```
Comment (Jennywh) [inline src/screens/LoginScreen.js:51]: keep the original error message.

File: src/screens/LoginScreen.js [hunk 2]
```diff
@@ -52,14 +61,14 @@ const LoginScreen = () => {
...
-    setError(''); // Clear any previous errors
+    setError(''); // Clear previous errors
 
     try {
       const userCredential = await signInWithEmailAndPassword(auth, email, password);
       const user = userCredential.user;
 
       await sendEmailVerification(user);
-      setError('Verification email resent. Please check your inbox and junk folder.');
+      setError('Verification email resent. Check your inbox and junk folder.');
```
Comment (Jennywh) [inline src/screens/LoginScreen.js:71]: keep original message

File: src/screens/SignUpScreen.js [hunk 3]
```diff
@@ -2,9 +2,8 @@ import React, { useState } from 'react';
 import { View, Text, TextInput, StyleSheet, TouchableOpacity, Alert } from 'react-native';
 import { Provider as PaperProvider } from 'react-native-paper';
 import { useNavigation } from '@react-navigation/native';
-// import auth, { createUserWithEmailAndPassword } from 'firebase/auth';
-import { getAuth, createUserWithEmailAndPassword, sendEmailVerification } from 'firebase/auth';
-const auth = getAuth();
+import { createUserWithEmailAndPassword, sendEmailVerification } from 'firebase/auth';
+import { auth } from '../../firebaseConfig';
```
Comment (Jennywh) [inline src/screens/SignUpScreen.js:6]: nice

**Review by Jennywh — APPROVED** [review]

**Pull request merged**